src/data/fintype/basic.lean

```diff
@@ -139,7 +141,13 @@ lemma not_mem_compl : a ∉ sᶜ ↔ a ∈ s := by rw [mem_compl, not_not]
 @[simp, norm_cast] lemma coe_compl (s : finset α) : ↑(sᶜ) = (↑s : set α)ᶜ :=
 set.ext $ λ x, mem_compl
 
-@[simp] lemma compl_empty : (∅ : finset α)ᶜ = univ := compl_bot
+lemma compl_empty : (∅ : finset α)ᶜ = univ := compl_bot
+
+lemma compl_univ : (univ : finset α)ᶜ = ∅ := compl_top
+
+@[simp] lemma compl_eq_empty_iff (s : finset α) : sᶜ = ∅ ↔ s = univ := compl_eq_bot
+
+@[simp] lemma compl_eq_univ_iff (s : finset α) : sᶜ = univ ↔ s = ∅ := compl_eq_top
```

Comment:
Lean will not use these lemmas to simplifty `(univ : finset α)ᶜ` and `(∅ : finset α)ᶜ`.

Reply:
(but these are still good simp lemmas!)